dash_docs/assets/override.css

```
@@ -150,6 +150,10 @@ h2, h3, h4, h5, h6 {
   border-radius: 1px;
 }
 
+.plotly a {
```

```suggestion
.js-plotly-plot .plotly a {
```

Probably doesn't matter here, but that's how we scope plotlyjs CSS internally for maximum specificity - so let's keep that for the sake of making this the best possible reference for our users.